discord/interactions.py

```diff
@@ -196,9 +196,6 @@ def _from_data(self, data: InteractionPayload):
         self._permissions: int = 0
 
         self._guild: Guild | None = None
-        self._guild_data = data.get("guild")
-        if self.guild is None and self._guild_data:
-            self._guild = Guild(data=self._guild_data, state=self._state)
```

Comment:
I didn't see it in payload so it probably shouldn't have been here earlier

Reply:
This change is 1) beyond the scope of this PR and 2) Incorrect, guild is included in Interaction payloads and acts as a fallback. Please revert.

Reply:
hm it was not included in types.Interactions.Interaction, but yea my bad